slippy/tile_grid.go

```diff
@@ -2,130 +2,165 @@ package slippy
 
 import (
 	"fmt"
-	"math"
 
 	"github.com/go-spatial/geom"
+	"github.com/go-spatial/proj"
 )
 
-// TileGrid contains the tile layout, including ability to get WGS84 coordinates for tile extents
-type Grid interface {
+// TileGriddor contains the tile layout, including ability to get WGS84 coordinates for tile extents
+type TileGriddor interface {
```

Comment:
question, why use the `or` suffix instead of `er`?

Reply:
I can never remember the rules in English for when to use 'or' va 'ed'. I choose 'or' because it sounded right, but i'm okay with 'ed'.

Reply:
Note `ed` but `er`!